packages/api/src/services/isPoolBlacklisted.ts

```diff
@@ -0,0 +1,11 @@
+const BLACKLISTED_POOL_IDS: string[] = [
```

Comment:
use a hashmap here will remove the O(n) check in fn.

```suggestion
const BLACKLISTED_POOL_IDS: Record<string, boolean> = [
  '421613_1_1689940800_v2': true,
  '421613_1_1689940801_v2': true,
];
```

Reply:
done